zanata-war/src/main/java/org/zanata/servlet/MDCInsertingServletFilter.java

```diff
@@ -39,6 +40,7 @@
  */
 @WebFilter(filterName = "MDCInsertingServletFilter")
 public class MDCInsertingServletFilter implements Filter {
+    private static final org.slf4j.Logger logReq = org.slf4j.LoggerFactory.getLogger("org.zanata.requests");
```

Comment:
`@Slf4j` won't let me name the field `logReq` - I want to make it clear that this is a non-standard logger (note the category): the subject is HTTP requests, not `MDCInsertingServletFilter`.  Plus I'm going off Lombok, it can be a bit fragile.